span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -126,6 +129,11 @@ public JaegerSpanNormalizer(Config config) {
     } else {
       this.tenantIdProvider = new DefaultTenantIdProvider(defaultTenantIdValue);
     }
+
+    this.tenantIdsToExclude =
+        config.hasPath(TENANT_IDS_TO_EXCLUDE_CONFIG)
+            ? config.getStringList(TENANT_IDS_TO_EXCLUDE_CONFIG)
+            : Collections.emptyList();
```

Comment:
Might want to log the set after the config initialization so that one time log stays in the log files.